core/src/main/java/org/apache/iceberg/FastAppend.java

```diff
@@ -114,6 +114,21 @@ public FastAppend appendManifest(ManifestFile manifest) {
     return this;
   }
 
+  @Override
+  public FastAppend toBranch(String branch) {
+    Preconditions.checkArgument(branch != null, "branch cannot be null");
+    if (ops.current().ref(branch) == null) {
+      super.createNewRef(branch);
+    }
+
+    Preconditions.checkArgument(ops.current()
+            .ref(branch).type()
+            .equals(SnapshotRefType.BRANCH),
```

Comment:
Does this check need to be split across 3 lines? Or is that autoformatting doing it?